learn/app-development/widgets/datalive/form/color-picker.md

```diff
@@ -9,16 +9,15 @@ id: ""
 
 <iframe width="100%" height="200" style="background-color: snow;" allowtransparency="true" src="https://apps.wavemakeronline.com/documentation_snippets/#/ColorPicker">Color Picker</iframe>
 
-# Properties
+## Properties
 
```

Comment:
Has desktop icon at a weird place. I think this was supposed to indicate that this is only available in web and not in mobile. We should find a better place for this icon.

![image](https://user-images.githubusercontent.com/57487155/69845743-affa4e80-1297-11ea-96d2-fb29d155e47e.png)

Reply:
as discussed we should evolve a patter for documenting where a widget can be used (desktop, mobile) and from which WM version it is available.

Reply:
Removed the icon and the duplicate file as well.